libzfs.pyx

```diff
@@ -1176,18 +1186,29 @@ cdef class ZFS(object):
         if cachefile:
             iargs.cachefile = cachefile
 
+        IF HAVE_ZPOOL_SEARCH_IMPORT_LIBZUTIL and HAVE_ZPOOL_SEARCH_IMPORT_PARAMS == 2:
+            cdef libzfs.libpc_handle_t lpch
+            lpch.lpc_lib_handle = self.handle
+            lpch.lpc_ops = &libzfs.libzfs_config_ops
+            lpch.lpc_printerr = True
```

Comment:
We may want to make this False in the future, since we are obtaining the error messages through the exception now.